lib/msf/util/exe.rb

```diff
@@ -1437,6 +1437,33 @@ def self.to_powershell_hta(framework, arch, code)
     read_replace_script_template("to_powershell.hta.template", hash_sub)
   end
 
+  def self.to_python_reflection(framework, arch, code, exeopts)
+    unless [ ARCH_X86, ARCH_X64, ARCH_AARCH64, ARCH_ARMLE, ARCH_MIPSBE, ARCH_MIPSLE, ARCH_PPC ].include? arch
+      raise RuntimeError, "Msf::Util::EXE.to_python_reflection is not compatible with #{arch}"
+    end
+    python_code = %(#{Rex::Text.to_python(code)}
```

Comment:
For maintainability, this should use a squiggly heredoc: https://github.com/rubocop-hq/ruby-style-guide#squiggly-heredocs. Also, moving to a one space indent (as in our other space-sensitive Python payloads) would save this format 20 characters, so I think it'd be worthwhile here as well.

Reply:
@acammack-r7 mind if I mark this as resolved, retest, and land?

Reply:
I fixed up the formatting, feel free to go for it!